doc/OVERMAP.md

```diff
@@ -241,16 +262,16 @@ completed--they are the "non-city" counterpart to the **city_building** type. Th
 made up of multiple overmap terrains (though not always), may have overmap connections (e.g. roads,
 sewers, subways), and have JSON-defined rules guiding their placement.
 
-### Mandatory Overmap Specials
+### Special placement
 
-There are a finite number of "slots" in which overmap specials can be placed during overmap
-generation, defined by the width of the overmap, height of the overmap, and an "overmap special
-frequency" (at the time of writing there are 72 "slots" per overmap). As a result, you are
-encouraged to exercise restraint when specifying some attributes of the overmap special, such as
-required minimum occurrences. The game gives precedence to "mandatory overmap specials" (e.g. those
-with a minimum greater than 0) and consequently too many mandatory overmap specials may exhaust the
-number of slots before any optional specials can even attempt placement. As a general rule, the
-minimum should be 0.
+There are a finite number of sectors in which overmap specials can be placed during overmap
+generation, each being a square with side length equal to `OMSPEC_FREQ` ( defined in omdata.h, at
+the time of writing `OMSPEC_FREQ`= 15 meaning each overmap has 144 sectors for placing specials). 
+At the beginning of overmap generation, a list of eligable map specials is built 
+( `overmap_special_batch` ). Next, a free sector is chosen where a special will be placed. A random
+point in that sector is checked against a random rotation of a special from the special batch to see
```

Comment:
Randomly chosen special?

Reply:
Yeah, random point in a random sector checked against a random rotation of a random special...very well randomized, all things considered. I'll update it to clear up potential confusion.